batch/batch/worker/worker.py

```diff
@@ -660,8 +679,16 @@ def __init__(self, batch_id, user, gsa_key, job_spec, format_version):
         containers = {}
 
         if input_files:
+            input_mounts = [{
+                'Source': LOCAL_SSD_MOUNT,
+                'Target': '/host',
+                'Type': 'bind',
+                'BindOptions': {
+                    'Propogation': 'shared'  # FIXME: look up if needed
```

Comment:
The questions are: 
- should Docker expose devices mounted inside LOCAL_SSD_MOUNT to the container
- should Docker expose devices mounted inside `/host` to the host OS
- should either of the above happen recursively, e.g. I mount something at LOCAL_SSD_MOUNT/foo and then I mount another thing at LOCAL_SSD_MOUNT/foo/foo2.